File: module-domain/src/main/java/eu/geekhome/domain/configurable/PowerLevelField.java
package eu.geekhome.domain.configurable;

import eu.geekhome.data.localization.Resource;

public class PowerLevelField extends FieldDefinition<Integer> {

    public PowerLevelField(String name, Resource hint, Validator<Integer>... validators) {
        super(name, hint, 0, Integer.class, new IntegerFieldBuilder(), validators);
    }
}

File: module-domain/src/main/java/eu/geekhome/domain/configurable/PowerLevelField.kt
package eu.geekhome.domain.configurable

import eu.geekhome.data.localization.Resource

class PowerLevelField(name: String, hint: Resource, vararg validators: Validator<Int>) :
    FieldDefinition<Int>(name, hint, 0, Int::class.java, IntegerFieldBuilder(), *validators)